app/src/main/java/com/techcourse/controller/IndexController.java

```diff
@@ -0,0 +1,17 @@
+package com.techcourse.controller;
+
+import com.interface21.context.stereotype.Controller;
+import com.interface21.web.bind.annotation.RequestMapping;
+import com.interface21.web.bind.annotation.RequestMethod;
+import com.interface21.webmvc.servlet.ModelAndView;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+@Controller
+public class IndexController {
+
+    @RequestMapping(value = "/", method = {RequestMethod.GET})
```

Comment:
`UserController`처럼 `@RequestMapping`의 method 중괄호가 없어도 괜찮겠네요!

```suggestion
    @RequestMapping(value = "/", method = RequestMethod.GET)
```

Reply:
네 그게 좋을 것 같네요!!

감사합니다!!